app/main.py

```diff
@@ -2,0 +1,16 @@
+def copy_file(command: str) -> None:
+
+    command_user = command.split(" ")
+
+    if len(command_user) < 3:
+        raise ValueError("Command must have at least 3 arguments")
+
+    if command_user[0] != "cp":
+        raise ValueError("Command must start with 'cp' to copy files")
+
+    file_src = command_user[1]
+    file_copy = command_user[2]
```

Comment:
Consider using more descriptive variable names instead of `file_src` and `file_copy`. For example, `source_file` and `destination_file` would make the code more readable.